docker/docker-compose-mainnet.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+  elrond-rosetta-observer-mainnet:
+    image: elrond-rosetta-observer-mainnet:latest
+    container_name: elrond-rosetta-observer-mainnet
+    ports:
+      - "${PORT_OBSERVER}:8080"
+      - "${PORT_P2P}:37373"
+    volumes:
+      - ${DATA_FOLDER}/node-${OBSERVED_SHARD}/logs:/data/logs
+      - ${DATA_FOLDER}/node-${OBSERVED_SHARD}/db:/data/db
+      - ${KEYS_FOLDER}:/keys
+    command: --destination-shard-as-observer=${OBSERVED_SHARD} --validator-key-pem-file=/keys/${OBSERVED_SHARD}.pem
+    networks:
+      elrond-rosetta-mainnet:
+        ipv4_address: 10.0.0.10
+
+  elrond-rosetta-online-mainnet:
+    image: elrond-rosetta:latest
+    container_name: elrond-rosetta-online-mainnet
+    ports:
+      - "${PORT_ROSETTA_ONLINE}:8080"
+    volumes:
+      - ${DATA_FOLDER}/rosetta/logs:/data/logs
+    command: --port 8080 --observer-http-url=http://10.0.0.10:8080 --observer-pubkey=${OBSERVER_PUBKEY} --chain-id=1 --num-shards=3 --observer-actual-shard=${OBSERVED_SHARD} --genesis-block=cd229e4ad2753708e4bab01d7f249affe29441829524c9529e84d51b6d12f2a7 --genesis-timestamp=1596117600 --native-currency=EGLD
```

Comment:
should we move the block hash and genesis timestamp to mainnet.env to make it similar to devnet.env